### DISCUSSION TITLE
render output to portrait 1080x1920

### DISCUSSION THREAD

**Discussion by ketut** [3b1b/manim#1588] [Q&A] [answered]
Dear all,
How to render output to portrait video (1080x1920)
Thank you

**Comment by TonyCrane** [comment, marked as answer]
When you render, you can add `-r 1080x1920` to the command to get the portrait video. But this does not change the size of the window (it is still horizontal). 
And constants.py stipulates that the ratio of the length of the horizontal axis to the length of the vertical axis is 16:9, so forcibly changing the screen ratio in this way will cause the unit length of the horizontal and vertical axes to be inconsistent (that is, the video is forcibly compressed horizontally to vertical). But you can change the aspect ratio by changing `ASPECT_RATIO` in constants.py.

But, to be honest, these methods are not easy. The easiest way is still to make a horizontal video (but only animate in the vertical range), and then edit it into a vertical video in the editing software.

**Reply by ketut** [reply]
Thank you @TonyCrane

**Comment by kwkr** [comment]
Hello! This page shows as one of the top results for the query `manim portrait` in Google. It looks though, like direct changes to the `constants.py` are no longer needed because I was able to achieve the results just by updating the config within my script like this:
```
config.frame_height = 14.222222222222221
config.frame_width = 8.0
config.frame_size = (1080,1920)
```
I also saw, there are also other ways of overriding the config. Is the method above obsolete or am I missing something?
